src/app/layout.tsx

```diff
@@ -1,11 +1,12 @@
 import type { Metadata } from "next";
 import { Inter } from "next/font/google";
+import "@/style/global.scss"
 
 const inter = Inter({ subsets: ["latin"] });
 
 export const metadata: Metadata = {
-  title: "Weather App",
-  description: "Weather application",
+  title: "Create Next App",
+  description: "Generated by create next app",
```

Comment:
Shingo changed the title and description here.
Could you please change it again?
Or if there is something you want, you can change it to whatever you want!

------------
  title: "Weather App",
  description: "Weather application",